bin/pygrb/pycbc_pygrb_page_tables

```diff
@@ -268,15 +249,28 @@ lower_dist = opts.lower_inj_dist
 num_bins = opts.num_bins
 wav_err = opts.waveform_error
 num_mc_injs = opts.num_mc_injections
-qf_outfile = opts.quiet_found_injs_output_file
-mf_outfile = opts.missed_found_injs_output_file
 lofft_outfile = opts.loudest_offsource_trigs_output_file
-lont_outfile = opts.loudest_onsource_trig_output_file
-qf_h5_outfile = opts.quiet_found_injs_h5_output_file
 lofft_h5_outfile = opts.loudest_offsource_trigs_h5_output_file
+lont_outfile = opts.loudest_onsource_trig_output_file
 lont_h5_outfile = opts.loudest_onsource_trig_h5_output_file
+qf_outfile = opts.quiet_found_injs_output_file
+mf_outfile = opts.missed_found_injs_output_file
+qf_h5_outfile = opts.quiet_found_injs_h5_output_file
 
-# Set output directories
+# Set output files and directories
+output_files = []
+if onsource_file is not None:
+    output_files = [lont_outfile, lont_h5_outfile, lofft_outfile,
+                    lofft_h5_outfile]
+    if len(output_files) != 4:
+        parser.error('Please provide both on-source output files ' +
+                     'and both off-source output file when using ' +
+                     '--onsource-file.')
+elif found_missed_file is not None:
+    output_files = [qf_outfile, mf_outfile, qf_h5_outfile]
+    if len(output_files) != 3:
+        parser.error('Please provide all 3 injections output files when ' +
+                     'using --found-missed-file')
```

Comment:
Same thing here. It looks like very basic Python lists count `None` entries in the `len()` function, but this could behave in a way I'm not expecting.